src/components/custom-media-library/CustomMediaLibraryModal.test.tsx

```diff
@@ -11,6 +11,8 @@ import { DopplerLegacyClient } from "../../abstractions/doppler-legacy-client";
 import { TestDopplerIntlProvider } from "../i18n/TestDopplerIntlProvider";
 
 const queryClient = new QueryClient();
+const SECONDS = 1000;
```

Comment:
Los segundos son estos, este tendria que valer 50, si efectivamente queres 50 segundos.

Reply:
si puede ser, para que sea mas deductivo a la hora de verse

Reply:
No, pero mas alla del nombre, el valor para SECONDS (o TIMEOUT_SECONDS) debe ser 50:
```suggestion
const SECONDS = 50;
```

Porque 1000 es la conversion a milisegundos.